docs/slack.rst

```diff
@@ -0,0 +1,36 @@
+Write the Docs Slack
+====================
+
+Our Slack channel has a bunch of interesting folks hanging out and chatting about documentation.
```

Comment:
I was going to comment precisely to suggest taking this opportunity to create additional channels on Slack so that people can actually follow the discussions without being overwhelmed with so many different conversations in #general.

I'd be interested in an "events" channel, in addition to the current "jobs" one, but I'm sure those more active in the #general channel will know of frequent topics which could benefit from dedicated discussion spaces.